### PR TITLE
update generated files in target folder

### PR DESCRIPTION
**Pull request by yatzek**

### CODE REVIEW

**Comment by simonjohansson** [comment]
Im not sure the target directory should be checked in, in the first place.

@mrdavidlaing ?

**Comment by mrdavidlaing** [comment]
@simonjohansson Actually, but convention we do checkin the `target/` folder; makes consuming the logstash-filters easier for downstream users since they don't have to get all the `bin/build` dependancies configured.

File: target/logstash-filters-default.conf [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # see https://github.com/cloudfoundry/loggregator/blob/master/src/loggregator/sinks/syslogwriter/syslog_writer.go#L156
 
 if [@type] in ["syslog", "relp"] and [syslog_program] == "doppler" {
-  # Parse Cloud Foundry logs from doppler (via https://github.com/cloudfoundry-community/firehose-to-syslog)
+  # Parse Cloud Foundry logs from doppler (via https://github.com/SpringerPE/firehose-to-syslog)
```
Comment (shinji62) [inline target/logstash-filters-default.conf]: Old.. repo..

**Comment by mrdavidlaing** [comment]
@yatzek - you seem to have built using an old version of the source.

For example, you are missing changed from https://github.com/logsearch/logsearch-for-cloudfoundry/commit/45041f4cfda4c082dcd1b749dc4b1320811171a8

Is your fork up to date?

**Comment by mrdavidlaing** [comment]
Closing, since this has been superceded by #67

**Pull request closed without merging**